common/tests/fixtures/cardano/get_address.json

```diff
@@ -0,0 +1,68 @@
+{
+  "setup": {
+    "mnemonic": "all all all all all all all all all all all all",
+    "passphrase": false
```

Comment:
how about using `passphrase: ""` and having unified code that always expects a PassphraseRequest?

Reply:
We can do that but all get_address tests here do not work with passphrases. So what's the advantage?

Reply:
the SLIP39 tests do have a passphrase

Reply:
I mean they do not check expected responses.

Reply:
ah, right, but using "passphrase is always string" simplifies that other thing that we wanted to do, converting the passphrase arg to `Optional[str]`

Reply:
I don't mind doing this but I still don't understand :). Passphrase is not passed anywhere in this case.

Reply:
`parametrize_using_common_fixtures` itself is using the field. There's a JSON that is using the passphrase in the very same set.

Basically let's assume that `passphrase` is always `str` in the JSONs, just to have more consistency

Reply:
> Basically let's assume that passphrase is always str in the JSONs, just to have more consistency

Ok, but let's discuss this with Suite later. Maybe they would like to distinguish false vs "".

Reply:
Let me take that back. Probably not since Suite turns on passphrase on default anyway. Added to my notes though

Reply:
Done in bbf842d193e4660fef32b2afe12f6196164df878